tests/common.sh

```diff
@@ -37,6 +37,28 @@ function wait_clean {
   done
 }
 
+# The argument is 'timeout in seconds'
+function get_uuid () {
+  counter=0
+  sleep_time=5
+  counter_max=$(( $1 / sleep_time ))
+  uuid="False"
+  until [ $uuid != "False" ] ; do
+    sleep $sleep_time
```

Comment:
If I'm following this right could we move the ```sleep $sleep_time``` into your if statement? Right now we always sleep X seconds before trying the first time. If we move it we might avoid a sleep.

Reply:
that makes sense. updated it